logpyle/__init__.py

```diff
@@ -644,6 +641,9 @@ def __init__(self, filename: Optional[str] = None, mode: str = "r",
 
         # }}}
 
+        import atexit
+        atexit.register(self.close)
```

Comment:
Make sure to unregister this if the `LogManager` is deleted before the program ends.

Reply:
Unfortunately, the atexit registration itself, as well as logging and warnings capture, prevented the logmgr instance from being deleted. I was able to work around it in [41de921](https://github.com/illinois-ceesd/logpyle/pull/146/commits/41de921c7b88cbebfe07bbcb968e1c5d0e6bdfcd).